lib/ansible/modules/system/java_certs.py

```diff
@@ -0,0 +1,250 @@
+  keystore_pass:
+    description:
+      - Keystore password
+    required: false
+    default: null
+  keystore_create:
+    description:
+      - Create keystore if it doesn't exist
+    required: false
+    default: null
+  state:
+    description:
+      - Defines action which can be either certificate import or removal.
+    choices: [ 'present', 'absent' ]
+    default: present
+    required: false
+
+author: Adam Hamsik @haad
+'''
+
+EXAMPLES = '''
+# Import SSL certificate from google.com to a given cacerts keystore
+java_certs:
+  cert_url: google.com
+  cert_port: 443
+  keystore_path: /usr/lib/jvm/jre7/lib/security/cacerts
+  keystore_pass: changeit
+  state: present
+
+# Remove certificate with given alias from a keystore
+java_certs:
+  cert_url: google.com
+  keystore_path: /usr/lib/jvm/jre7/lib/security/cacerts
+  keystore_pass: changeit
+  state: absent
+
+# Import SSL certificate from google.com to a keystore, create it if it doesn't exist
+java_certs:
+  cert_url: google.com
+  keystore_path: /tmp/cacerts
+  keystore_pass: changeit
+  keystore_create: yes
+  state: present
+'''
+
+RETURN = '''
+msg:
+  description: Output from stdou of keytool command after execution of given command.
+  returned: success
+  type: string
+  sample: "Module require existing keystore at keystore_path '/tmp/test/cacerts'"
+
+rc:
+  description: Keytool command execution return value
+  returned: success
+  type: int
+  sample: "0"
+
+cmd:
+  description: Executed command to get action done
+  returned: success
+  type: string
+  sample: "keytool -importcert -noprompt -keystore"
+'''
+
+import os
+
+def check_cert_present(module, keystore_path, keystore_pass, alias):
+    ''' Check if certificate with alias is present in keystore located at keystore_path '''
+    test_cmd = ("keytool -noprompt -list -keystore '{}' -storepass '{}' "
+                "-alias '{}'").format(keystore_path, keystore_pass, alias)
+
+    (check_rc, _, _) = module.run_command(test_cmd)
+    if check_rc == 0:
+        return True
+    return False
+
+def import_cert_url(module, url, port, keystore_path, keystore_pass, alias):
+    ''' Import certificate from URL into keystore located at keystore_path '''
+    fetch_cmd = ("keytool -printcert -rfc -sslserver {}:{}").format(url, port)
+    import_cmd = ("keytool -importcert -noprompt -keystore '{}' -storepass '{}' "
+                  "-alias '{}'").format(keystore_path, keystore_pass, alias)
+
+    # Fetch SSL certificate from remote host.
+    (_, fetch_out, _) = module.run_command(fetch_cmd, check_rc=True)
+
+    # Use remote certificate from remote host and import it to a java keystore
+    (import_rc, import_out, import_err) = module.run_command(import_cmd, data=fetch_out,
+                                                             check_rc=False)
+    if import_rc == 0:
+        return module.exit_json(changed=True, msg=import_out,
+                                rc=import_rc, cmd=import_cmd, stdout_lines=import_out)
```

Comment:
If you want to return the stdout, use `stdout` key.
Ansible will automatically add `stdout_lines` key for you.

> When `stdout` is returned, Ansible always provides a list of strings, each containing one item per line from the original output.

http://docs.ansible.com/ansible/common_return_values.html#stdout-lines

(So do L156 & L170)